### ISSUE TITLE
term-dmenu evaluates output

### ISSUE THREAD

**Issue by calbrecht** [Seirdy/term-dmenu#2]
This is a stripped down version with abduco and stderr redirects removed, but basically the same as `term-dmenu`.

`export input='$(echo hi, i get evaluated)'; dash -c 'output=$(echo "$input" | fzf); export output; dash -c "echo \"$output\""'`

I noticed this behaviour when using `rg` output as input to `fzf` in `term-dmenu`.

**Comment by RayZ0rr** [comment]
Yes, I was looking for this fix. It would also be cool to make j4-dmenu-desktop optional in the sense that it'll be used if it exists.

**Comment by RayZ0rr** [comment]
@calbrecht Would it also be possible to use bash instead of dash? Bash is in pretty much all systems I use so if possible I would like to avoid installing dash.
